Directory.Build.props

```diff
@@ -41,7 +31,7 @@
    -->
   <PropertyGroup>
     <NewtonsoftVersion>8.0.3</NewtonsoftVersion>
-    <NewtonsoftVersion Condition=" $(VsTargetVersion) == '2022' " >13.0.1</NewtonsoftVersion>
```

Comment:
Miscellaneous: whitespace inside the tag wasn't reserved. Same in line 55 below. However, it's very minor, and I don't think there is anything we can do about it anyway.